inline-cli.py

```diff
@@ -26,14 +29,14 @@ def parse(filename):
         for line in infile:
             # new filename
             if not line.startswith(' '):
-                add_message(messages, current_filename, current_line, current_message_content)
```

Comment:
Line: 40 - pylint: redefined-variable-type / Redefinition of current_line type from str to int (col 16)